eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -123,7 +123,7 @@ impl Eth2NearRelay {
     }
 
     fn block_known_on_near(&self, slot: u64) -> Result<bool, Box<dyn Error>> {
-        trace!(target: "relay", "Check if block with slot={} on NEAR", slot);
+        debug!(target: "relay", "Check if block with slot={} on NEAR", slot);
```

Comment:
This is should be a `trace` level message
```suggestion
        trace!(target: "relay", "Check if block with slot={} on NEAR", slot);
```

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/775/commits/e7455a096a6599199ee548c186f66717effc1963